CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.3.1-wip
```

Comment:
Pretty sure we'll have to make this v2.0.0 – since it's a breaking change

Reply:
Done!